dxcherry/src/github/createPullRequest.ts

```diff
@@ -1,66 +1,19 @@
-import { getClient } from './createClient';
-import { getUser } from '../github/getters';
-import { getBranchName } from '../git/branch';
-import { REPO_DATA } from './config';
-import log from '../log';
-import { window } from 'vscode';
+import GithubClient from './client';
+import { logInfo } from '../info';
 
-async function setLabels (issueNumber: number, labels: string[]) {
-  await getClient().issues.setLabels(Object.assign({}, REPO_DATA, { issue_number: issueNumber, labels }));
-};
-
-async function addAssign (issueNumber: number, assignees: string[]) {
-  await getClient().issues.addAssignees(Object.assign({}, REPO_DATA, { issue_number: issueNumber, assignees }));
-};
-
-async function addReviewers (pullNumber: number, reviewers: string[]) {
-  await getClient().pulls.requestReviewers(Object.assign({}, REPO_DATA, { pull_number: pullNumber, reviewers }));
-};
-
-async function createPullRequest ({ title, description, versions, reviewers, labels }: any) {
-  log.appendLine('start creating pull requests');
-  const { login } = await getUser();
-  const branch = await getBranchName();
-  const branchWithoutVersion = branch.match('(.*)_[0-9]*_[0-9]*')[1];
-
-  log.appendLine(`login: ${login}, branch: ${branch}, branchWithoutVersion: ${branchWithoutVersion}`);
-
-  versions.forEach(async (version: string, index: number) => {
-    const payload = Object.assign({},
-      REPO_DATA, {
-        base: version,
-        head: `${login}:${branchWithoutVersion}_${version}`,
-        title,
-        body: description
-      });
-    const errMessage = `Pull request from ${payload.head} to ${version} was not created. Please check, that ${payload.head} was pushed to the fork.`;
+async function createPullRequest (login: string, branchWithoutVersion: string, version: string, title: string, description: string, labels: string[], reviewers: string[]) {
```

Comment:
7 arguments?

Reply:
yeap, I doubt about it. Do u think it's better to use object parameter? It'll just add `{ }` in two rows

Reply:
oh, fixed it